app/src/store/voice/accessToken.ts

```diff
@@ -60,19 +60,15 @@ export const getAccessToken = createTypedAsyncThunk<
 
   const tokenTextResult = await settlePromise(tokenResponse.text());
 
-  if (!tokenResponse.ok && tokenTextResult.status === 'fulfilled') {
-    return rejectWithValue({
-      reason: 'TOKEN_RESPONSE_NOT_OK',
-      statusCode: tokenResponse.status,
-      error: miniSerializeError(tokenTextResult.value),
-    });
-  }
-
   if (!tokenResponse.ok) {
+    const error =
+      tokenTextResult.status === 'fulfilled'
+        ? tokenTextResult.value
```

Comment:
What does the compiler say the value of `tokenTextResult.value` is here? I think it's a `string`, so we should probably wrap it in an error too.